.github/workflows/linting.yml

```diff
@@ -22,7 +22,8 @@ jobs:
     - name: install Python packages
       run: |
         pip install --upgrade pip
-        pip install --upgrade flake8
+        # fix to this version for develop branch (to avoid needing to fix geant4.py)
+        pip install --upgrade "flake8<7.1"
```

Comment:
We'll sync `develop` to `5.0.x` eventually, so this will also land in there (unless we kick it out when doing sync), but OK for now...